python/dgl/heterograph.py

```diff
@@ -4751,6 +4791,8 @@ def is_unibipartite(graph):
         True if the graph is a uni-bipartite.
     """
     src, dst, _ = graph.edges()
+    print(src.tonumpy())
+    print(dst.tonumpy())
```

Comment:
I assume these two lines are for debugging and should be removed?

Reply:
Yes.